src/main/java/com/greedobank/cards/model/CardTemplate.java

```diff
@@ -0,0 +1,61 @@
+package com.greedobank.cards.model;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import com.greedobank.cards.utils.CardType;
+import com.greedobank.cards.utils.CurrencyType;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.time.OffsetDateTime;
+import java.util.Set;
+
+@Entity
+@Data
```

Comment:
are you sure you want to have `toString` and stuff generated for Entities?
The same for all entities

Reply:
Done